python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py

```diff
@@ -170,14 +166,18 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
     assert result.success
 
+
+def test_asset_checks_are_logged(mocked_context, dbt_commands: List[List[str]]):
```

Comment:
Something to suggest that this is a resource level test

```suggestion
def test_asset_checks_are_logged_from_resource(mocked_context, dbt_commands: List[List[str]]):
```